x/ccv/provider/keeper/throttle.go

```diff
@@ -574,7 +576,7 @@ func (k Keeper) GetSlashMeter(ctx sdktypes.Context) sdktypes.Int {
 // SetSlashMeter sets the slash meter to the given signed int value
 //
 // Note: the value of this int should always be in the range of tendermint's [-MaxTotalVotingPower, MaxTotalVotingPower]
-func (k Keeper) SetSlashMeter(ctx sdktypes.Context, value sdktypes.Int) {
+func (k Keeper) SetSlashMeter(ctx sdktypes.Context, value math.Int) {
```

Comment:
Does marshaling `math.Int` give the same result as marshaling `sdktypes.Int`?

Reply:
`sdktypes.Int` is an alias for `math.Int`
- https://github.com/cosmos/cosmos-sdk/blob/666c345ad23ddda9523cc5cd1b71187d91c26f34/types/math.go#L15

No action necessary